controllers/contacts/updateContact.js

```diff
@@ -0,0 +1,41 @@
+const { NotFound } = require("http-errors");
+
+const { contactSchema } = require("../../schemas");
+
+const contactsOperations = require("../../models/index");
+
+const updateContact = async (req, res, next) => {
+  try {
+    const { error } = contactSchema.validate(req.body);
+    if (error) {
+      res.status(400).json({
+        status: "error",
+        code: 400,
+        message: `missing  field ${error.message}`,
+      });
+      return;
+    }
```

Comment:
```suggestion
    const { error } = contactSchema.validate(req.body);
    if (error) {
      res.status(400).json({
        status: "error",
        code: 400,
        message: `missing  field ${error.message}`,
      });
      return;
    }
```
У вас в схемі усі поля обов'язкові. А має бути можливість оновити тільки одне поле.